requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ django-ratelimit-backend==0.6
 unicodecsv==0.9.4
 django-require==1.0.6
 pyuca==1.1
+wrapt==1.10.5
```

Comment:
@utkjad can you also provide some context on what this requirement is and why we need it?  Is it used in production or just during dev/test? Any other things about the library we should know?